sklearn/impute/tests/test_impute.py

```diff
@@ -1500,3 +1498,44 @@ def test_most_frequent(expected, array, dtype, extra_value, n_repeat):
     assert expected == _most_frequent(
         np.array(array, dtype=dtype), extra_value, n_repeat
     )
+
+
+def test_impute_one_feature():
```

Comment:
As this is a non-regression test for #19352, we can make a reference to it in the test docstring.
I, here, also suggest another name for the test.

```suggestion
def test_pls_regression_iterative_imputer():
    """IterativeImputer should support PLSRegression as an estimator.

    Non-regression test for issue #19352.
    """
```